modules/python/test/test_copytomask.py

```diff
@@ -24,12 +24,13 @@ def test_copytomask(self):
         valeurBGRSup = np.array([70, 70,255])
         maskRed = cv.inRange(img, valeurBGRinf, valeurBGRSup)
         #New binding
-        dstcv  = cv.copyTo(img,maskRed)
+        dstcv = np.full_like(img, 255)
+        cv.copyTo(img, dstcv, maskRed)
```

Comment:
`dstcv = cv.copyTo(img, mask=maskRed)` ?

Reply:
this will fail as dstcv will be allocated in copyTo.